src/main/resources/hudson/plugins/git/util/BuildData/index.jelly

```diff
@@ -8,8 +8,18 @@
 	<l:main-panel>
 	<h1>${%Git Build Data}</h1>
 
-	<strong>${%Revision}:</strong> ${it.lastBuild.SHA1.name()}
-        <j:if test="${!empty(it.scmName)}"> from <strong>SCM:</strong> ${it.scmName}</j:if>
+	<strong>${%Revision}</strong>: ${it.lastBuild.SHA1.name()}
+	<j:if test="${!empty(it.scmName)}"><br/><strong>${%SCM}</strong>: ${it.scmName}</j:if>
```

Comment:
this change breaks the ATH test.
https://ci.jenkins.io/job/Core/job/acceptance-test-harness/job/master/335/testReport/plugins/GitPluginTest/java_8_split9___custom_scm_name/history/

as the `:` is now outside the `strong` tag

Reply:
jenkinsci/acceptance-test-harness#666

Reply:
Thanks for reporting and for the investigation.  Is it enough to put the ':' back inside the `strong` tag?

Reply:
that would work - but I am equally fine with the fix to the ATH to adapt the behaviour!